fineract-provider/build.gradle

```diff
@@ -688,6 +582,12 @@ dependencies {
 cargo {
     containerId "tomcat9x"
 
+    // looks like Cargo doesn't detect the WAR file automatically in the multi-module setup
+    deployable {
+        file = file("${buildDir}/libs/fineract-provider-${releaseVersion}.war")
+        context = 'fineract-provider'
```

Comment:
As per JIRA-1187 ticket, we should keep the name of the war file as "fineract-provider.war" without the release version. Otherwise this will confuse everyone who tries to deploy it to Tomcat and finds that the URL to access Fineract keeps changing every time there's a new version...

Reply:
Can we take care of that separately? Because this is already in place upstream and it would force me to touch the distribution configuration and the release configuration... which would be more than needed to make this work here. But I can take care of this once we are done with multi-module (or squeeze it in between with a separate PR). Good for you?

Reply:
Hmm... not sure I understand. After FINERACT-1187 was merged 4 days ago, we were building fineract-provider.war. Are you saying that some other change has now broken that again, and we are back to building fineract-provider-releaseVersion.war?

Reply:
But wouldn't that be a "regression" of FINERACT-1187? Why do we have to break that here?

Reply:
I was not aware of that change... that's all. If you say that this is already working in develop then I have to pull in the recent changes; that's all.

Is FINERACT-1187 already merged?

Reply:
FINERACT-1187 was merged and working. It's change to build.gradle which is part of this PR that is breaking it again. I've marked the problematic line above with a review comment.

Reply:
And of course once you fix that problem in build.gradle, then this change to cargo is not needed...

Reply:
To know if fixes are merged, check that Links To section thingie on the FINERACT-1187 JIRA... notice the link to #1379 - makes sense? In general, any issues in Status Resolved,  Resolution: Fixed should have associated Links To PRs... if they don't, we did something wrong.